src/autopas/pairwiseFunctors/CellFunctor.h

```diff
@@ -177,7 +180,7 @@ template <class Particle, class ParticleCell, class ParticleFunctor, DataLayoutO
 template <bool newton3>
 void CellFunctor<Particle, ParticleCell, ParticleFunctor, DataLayout, useNewton3, bidirectional>::processCellAoS(
     ParticleCell &cell) {
-  if (cell.size() > _startSorting) {
+  if (cell.size() > _sortingThreshold and _useSorting) {
```

Comment:
please move the check for `_useSorting` to the front in all those `if`s. For two reasons: first, it is the easiest and most predictable to check, thus improving branching performance. Second, it makes the code much more intuitive to read because you immediately state what the if is about.

Reply:
done in [8081014](https://github.com/AutoPas/AutoPas/pull/619/commits/80810142d34cf7872346976089a90e1753180bb7)